Buildings/Controls/OBC/ASHRAE/G36/Generic/TrimAndRespond.mo

```diff
@@ -377,9 +441,40 @@ In other words, every time step <code>samplePeriod</code>:
 <img alt=\"Image of set point reset\"
 src=\"modelica://Buildings/Resources/Images/Controls/OBC/ASHRAE/G36/Generic/TrimRespond.png\"/>
```

Comment:
Can you please make three individual figures. Otherwise it is so wide that it is hard to see in tools or the html output.

Reply:
Addressed in abc80d6684